### PR TITLE
Updates blueprint to latest

### PR DESCRIPTION
**Pull request by scalvert**
Updates blueprint to `3.27.0-beta.1`.

- Updates dummy app to use Octane syntax
- Updates tests to remove `EmberObject` usage
- Removes deprecated APIs slated for removal in 7.0.0.

🚨 breaking changes
- Drops support for Node 10
- Removes `disposable` APIs (targeted for removal in 7.0.0)
- Removes DOM APIs (targeted for removal in 7.0.0)

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
....js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/app/routes/foo.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/config/environment.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/dummy/config/targets.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/destroy-app.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/module-for-acceptance.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/resolver.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/helpers/start-app.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/test-helper.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/debounce-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/dom-event-listeners-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/poll-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/run-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/tests/unit/utils/get-task-test.js",[],"/Users/scalvert/workspace/personal/ember-lifeline/types/dummy/index.d.ts",[]]
```
Comment (rwjblue) [inline .eslintcache:1]: This should not be committed (should be added to `.gitignore`)

File: hound.yml [hunk 2]
```diff
@@ -1,4 +0,0 @@
-eslint:
```
Comment (rwjblue) [inline hound.yml:1]: Did you want to delete this?

File: tests/index.html [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
-    {{content-for "head"}}
-    {{content-for "test-head"}}
+    {{content-for "head"}} {{content-for "test-head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/test-support.css">
+    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css" />
+    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css" />
+    <link rel="stylesheet" href="{{rootURL}}assets/test-support.css" />
```
Comment (rwjblue) [inline tests/index.html:14]: Closing these tags is odd (its XHTML not HTML)

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: .eslintcache (hunk 1 above)
Comment (scalvert) [reply .eslintcache:1]: Ya not sure why this snuck in.

File: hound.yml (hunk 2 above)
Comment (scalvert) [reply hound.yml:1]: Yes.

File: tests/index.html (hunk 3 above)
Comment (scalvert) [reply tests/index.html:14]: This was an auto-editor thingy. I can revert.

File: .eslintcache (hunk 1 above)
Comment (scalvert) [reply .eslintcache]: Fixed.

File: tests/index.html (hunk 3 above)
Comment (scalvert) [reply tests/index.html]: Fixed.

File: addon/run-task.ts [hunk 4]
```diff
@@ -229,6 +229,7 @@ export function throttleTask(
   let timers: Set<EmberRunTimer> = getTimers(destroyable);
   let cancelId: EmberRunTimer = throttle(
     destroyable as any,
+    // @ts-ignore
```
Comment (scalvert) [inline addon/run-task.ts:232]: This was to avoid the type mismatch in throttle.

Comment (rwjblue) [reply addon/run-task.ts:232]: ya, makes sense (might be good to throw a comment in there so we remember)

Comment (scalvert) [reply addon/run-task.ts:232]: Will do.

**Pull request closed without merging**